lib/class-model.js

```diff
@@ -599,6 +599,12 @@ collect_matching_class_methods (ArtClassVisitor * self,
     }
     else
     {
+      const gchar * name_begin;
+
+      name_begin = bare_method_name + class_name_length + 1;
+      if (is_constructor && g_str_has_prefix (name_begin, "<clinit>"))
```

Comment:
Does the string contain anything after `<clinit>`?

Reply:
Nope.